docs/publishing/index.md

```diff
@@ -0,0 +1,123 @@
+- _Optional_: [use `type: "module"` when possible](#optional-use-type-module)
+- _Optional_: [include typings; e.g. `*.d.ts` files](#optional-include-typings)
+- _Optional_: [include source maps](#optional-include-source-maps)
+
+### Don't: 👎
+
+- ❌ [Do not bundle](#do-not-bundle)
+- ❌ [Do not minify](#do-not-minify)
+- ❌ [Do not optimize](#do-not-optimize)
+- ❌ [Do not use `.mjs` file extensions](#do-not-use-mjs-file-extensions)
+- ❌ [Do not import polyfills](#do-not-import-polyfills)
+
+## Do publish latest standard EcmaScript
+
+Contrary to some long-standing beliefs, an implementing project will always know more about the delivery context of the code it leverages than the developer of any of its dependencies. Taking this fact into account, the act of transpiling your code below a reasonably modern level (e.g. [TC39 Stage 4](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and/or available cross-browser) is the best way that you can prepare your package to empower downstream projects to make their own decisions about the code they ship to their users.
+
+In some cases, you may choose to use newer specifications that may require special care when preparing for older browsers. In this case, transpiling down to the level you choose from above may be a good idea. If you choose not to do this, be sure to document this reality thoroughly so that your users are not left unawares.
+
+## Do publish standard es modules
+
+Your package (and those packages that it depends on) should be distributed as standard es modules. This supports the "do" above to deliver the latest standard EcmaScript to your users, modules are both Stage 4 and supported in all major browsers, but also prepares your users to have the simplest time building their applications for production as es modules are the only module format natively supported by browsers.
+
+## Do include appropriate `package.json` entries
+
+Your `package.json` should both its `main` entrypoint and `module` entrypoint to the same es module file. You will find this already configured for you by `npm init @open-wc`, but ensure your `package.json` file includes something simialar to the following:
+
+```
+{
+  "main": "index.js",
+  "module": "index.js",
+  // ...
+}
+```
+
+## Do export element classes
+
+Every custom element is a class extension of `HTMLElement`; e.g. `class MyElement extends HTMLElement {}`. Be sure that you export this class from your package. When starting from `npm init @open-wc` you will see this export in `src/MyElement.js` and reported by the `index.js` entry point. This will make it possible for both customization of your element via extension (e.g. `class NewElement extends MyElement {}`) as well as for advanced usage of the custom element in Scoped Registries, both [native](https://github.com/w3c/webcomponents/issues/716) (when available) or [synthetic](/scoped-elements/).
```

Comment:
```suggestion
Every custom element is a class extension of `HTMLElement`; e.g. `class MyElement extends HTMLElement {}`. Be sure that you export this class from your package. When starting from `npm init @open-wc` you will see this export in `src/MyElement.js` and referenced by the `index.js` entry point. This will make it possible for both customization of your element via extension (e.g. `class NewElement extends MyElement {}`) as well as for advanced usage of the custom element in Scoped Registries, both [native](https://github.com/w3c/webcomponents/issues/716) (when available) or [synthetic](/scoped-elements/).
```